apps/spruce/src/pages/adminSettings/tabs/OtherTab/OtherTab.tsx

```diff
@@ -0,0 +1,6 @@
+export const WebTab = () => (
```

Comment:
A bunch of these exports are named WebTab

Reply:
removing the ones not rendered